module/zfs/abd.c

```diff
@@ -251,6 +251,27 @@ abd_chunkcnt_for_bytes(size_t size)
 #define	__GFP_RECLAIM		__GFP_WAIT
 #endif
 
+#define	ABD_FILE_CACHE_PAGE	0x2F5ABDF11ECAC4E
```

Comment:
You're going to need to define a smaller value for 32-bit platforms to resolve the build failures on arm/ppc/i386.

```
/var/lib/buildbot/slaves/buildslave1/Debian_8_arm__BUILD_/build/zfs/module/zfs/abd.c: In function ‘abd_mark_zfs_page’:
include/linux/mm.h:319:54: error: large integer implicitly truncated to unsigned type [-Werror=overflow]
 #define set_page_private(page, v) ((page)->private = (v))
                                                      ^
/var/lib/buildbot/slaves/buildslave1/Debian_8_arm__BUILD_/build/zfs/module/zfs/abd.c:264:2: note: in expansion of macro ‘set_page_private’
  set_page_private(page, ABD_FILE_CACHE_PAGE);
```